cnxarchive/tests/test_views.py

```diff
@@ -1444,6 +1469,27 @@ def test_exports_without_version(self):
         with self.assertRaises(IdentHashMissingVersion) as cm:
             get_export(self.request)
 
+    def test_get_extra_404(self):
+        #id = 'b771c6fc-34f0-11e7-ad77-e3343f783f02'
+        #version = '1.1'
+        id = '94919e72-7573-4ed4-828e-673c1fe0cf9b'
+        version = '66.1'
+
+        # Build the request
+        self.request.matchdict = {'ident_hash': '{}@{}'.format(id, version)}
+        self.request.matched_route = mock.Mock()
+        self.request.matched_route.name = 'content-extras'
+
+        from ..views import get_extra
+
+        # self.assertRaises(httpexceptions.HTTPNotFound,
+        #                   get_extra, self.request)
+
+        with self.assertRaises(httpexceptions.HTTPNotFound) as caught_exc:
+            import pdb;
```

Comment:
Addressed in https://github.com/Connexions/cnx-archive/pull/477/commits/4aa2b25d8b68607229d228d30fe195fd45aeb69d